DataConnectors/ZPA/Connector_LogAnalytics_agent_Zscaler_ZPA.json

```diff
@@ -0,0 +1,166 @@
+{
+    "id": "Zscaler",
```

Comment:
Could you please change the connector id , As we already has the connector deployed with same ID.

Thanks,
Aditi Maheshwari

Reply:
updated